src/AddTodoForm.js

```diff
@@ -1,18 +1,33 @@
-import React from 'react';
+import React, { useState } from 'react';
+
+const AddTodoForm = ({ onAddTodo }) => { // Destructuring the props
+    const [todoTitle, setTodoTitle] = useState(''); 
+
+    const handleTitleChange = (event) => {
+        const newTodoTitle = event.target.value;
+        setTodoTitle(newTodoTitle);
+    };
 
-const AddTodoForm = ({ onAddTodo }) => {
     const handleAddTodo = (event) => {
-        event.preventDefault(); //prevent form from refreshing
-        const todoTitle = event.target.title.value;
-        onAddTodo(todoTitle);
-        event.target.reset();   //reset the form after submission
-        console.log(todoTitle);
+        event.preventDefault();
+        const newTodo = {
+            title: todoTitle,
+            id: Date.now(), // generate unique identifier
+        };
+        onAddTodo(newTodo); 
```

Comment:
If you want to clean up your code. How else can you pass your todo object? This works just fine so if you don't have time you can merge without changing this.